### DISCUSSION TITLE
🔦 Feature: Inquiry about the completeness and openness of the nc-help library

### DISCUSSION THREAD

**Discussion by shaneyxs** [nocodb/nocodb#10329] [General]
### Please confirm if feature request does NOT exist already ?

- [X] I confirm there is no existing issue for this

### Describe the usecase for the feature

Title: Inquiry about the completeness and openness of the nc-help library
Description
Hi NocoDB Team,

First, I would like to thank you for the amazing work on NocoDB.  It has been an invaluable tool for my projects.

I am writing to inquire about the nc-help library within NocoDB.  It seems that the library is not fully developed or entirely open.  As a user who relies on detailed command usage and help documentation, I am interested in understanding the following:

Is there a specific reason why the nc-help library is not fully open or complete?
Are there any plans to develop and expand the nc-help library in the near future?
Can the community contribute to the development of the nc-help library?  If so, what guidelines should we follow?
Understanding these points would help users like me better utilize NocoDB and potentially contribute to its improvement.

Thank you for your time and assistance.

### Suggested Solution

Skip over

**Comment by samuellie** [comment]
any update regarding this? would like to understand about this too

**Comment by o1lab** [comment]
Nc-help is no longer used and all code is available in nocodb package. We didn't want contributions initially and then there were devs who were abusing posthog key available in code and racking up 100s of dollar worth of events. This made us move posthog part to nc-help. That too didn't help. So it's all out there and we just live with it.
